src/test/resources/com/tresys/nitf/xsd/nitf_ude_types.dfdl.xsd

```diff
@@ -0,0 +1,156 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I wonder if this we should split this repo into two sub projects, something like nitf and nitf-core? nitf-core would have the current nitif.dfdl.xsd file? And then nitf would contain this file and all the nitf tests? That way, nitf can depend on nitf-core and work the way it does now, and variations like raster can depend on nitf-core and implement the necessary extension types for added functionality?